addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6555,274 +6555,176 @@ stock GetBossIndex(client)
 
 stock ParseFormula(client, const String:key[], const String:defaultFormula[], defaultValue)
 {
-	decl String:formula[1024], String:buffer[2];
-	new String:strValue[1024], Float:sum[128], calcOperators:_operator[128];
-	new bool:closeBracket, bool:usePlayers, bool:foundOperator, bracket;
+	decl String:formula[1024];
 	KvRewind(BossKV[Special[client]]);
 	KvGetString(BossKV[Special[client]], key, formula, sizeof(formula), defaultFormula);
 	ReplaceString(formula, sizeof(formula), " ", ""); //Get rid of spaces
-	new length=strlen(formula);
-	for(new i; i<=length; i++)
+    Float:indexes[2] = {0, strlen(formula)};
+	new result=RoundFloat(DoMiniOperation(formula, indexes, BossKV[Special[client]], key));
```

Comment:
I meant the entire function.  There's no need to have a `DoMiniOperation` that only gets called once.